### PR TITLE
Add video to iOS deploy

### LINKED ISSUE #9466: Embed video into releasing an ios app
### What information is missing?

Can we embed this video https://www.youtube.com/watch?v=iE2bpP56QKc into this page https://docs.flutter.dev/deployment/ios? 

### How would you like us to fix this problem?

_No response_

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #9466

### CODE REVIEW

**Review by leighajarett — APPROVED** [review]
LGTM, thank you for adding this!

**Review by parlough — APPROVED** [review]
Thanks for adding this embed and thanks @leighajarett for the great video!

Two small suggestions:

**Review by domesticmouse — APPROVED** [review]
LGTM once @parlough's issues are dealt with =)

File: src/deployment/ios.md [hunk 1]
```diff
@@ -16,13 +16,17 @@ Before beginning the process of releasing your app,
...
-In order to publish your app to the App Store,
+To publish your app to the App Store,
 you must first enroll in the
 [Apple Developer Program][devprogram].
 You can read more about the various
 membership options in Apple's
 [Choosing a Membership][devprogram_membership] guide.
 
+## Video overview
+
```
Comment (atsansone) [inline src/deployment/ios.md:27]: **nitpick:** 

```suggestion

For those who prefer video over text,
the following video covers the same material as this guide.
```

**Pull request merged**